### PR TITLE
fix(ScrollViewer): Content margin was not taken into consideration for Skia and WASM.

### LINKED ISSUE #8213: [iOS] ScrollViewer_Margin test is failing
### Current behavior

All builds are failing since https://github.com/unoplatform/uno/pull/8098 (last successful build: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=41570&view=results).

Seems the test `ScrollViewer_Margin` is now flaky as on iOS changing `Height` then `Width` works but changing `Width` then `Height` fails.

Test is has been disabled for now on iOS: https://github.com/unoplatform/uno/commit/c971d8e503c42a81ee94a9057a7efd9f37d29cb8

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

_No response_

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
(Note for WASM; bottom or right margin in ScrollViewer cannot be scrolled to the end still need to be fixed with issue #7000)

GitHub Issue (If applicable): #7000,  #6443, resolves #8213

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Content margin was not taken into consideration for Skia and WASM.
(Note for WASM; bottom or right margin in ScrollViewer cannot be scrolled to the end still need to be fixed with issue #7000)

## What is the new behavior?

Content margin is taken into consideration for Skia entirely and the top|left margin for WASM.
(Note for WASM; bottom or right margin in ScrollViewer cannot be scrolled to the end still need to be fixed with issue #7000)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [X] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by dr1rrb — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs [hunk 1]
```diff
@@ -721,52 +721,82 @@ void UpdateDimensionProperties()
...
-					ExtentWidth = canUseActualWidthAsExtent ? fe.ActualWidth : fe.DesiredSize.Width;
+					extentWidth = canUseActualWidthAsExtent ? fe.ActualWidth : fe.DesiredSize.Width;
 				}
+
+#if __WASM__
+				// Issue needs to be fixed first for WASM for Right Margin missing
+				// Details here: https://github.com/unoplatform/uno/issues/7000
+				ExtentWidth = extentWidth + fe.Margin.Left;
+#else
+				ExtentWidth = extentWidth + fe.Margin.Left + fe.Margin.Right;
```
Comment (MartinZikmund) [inline src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs:767]: As this is still a problem - should #7000 be closed with this PR?

Comment (agneszitte-nventive) [reply src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs:767]: Correctly updated the description of the issue, thanks ;) !

**Comment by MartinZikmund** [comment]
@agneszitte-nventive is this ready 😊 ?

**Review by MartinZikmund — APPROVED** [review]
🚀🚀🚀

**Pull request merged**